src/data/mv_polynomial/rename.lean

```diff
@@ -243,4 +243,25 @@ end
 
 end coeff
 
+section support
+
+lemma support_rename_of_injective {p : mv_polynomial σ R} {f : σ → τ} (h : function.injective f) :
+  (rename f p).support = finset.map (map_domain_embedding ⟨f,h⟩) p.support :=
```

Comment:
This lemma might be easier to use if the RHS is about `finset.image` and avoids `map_domain_embedding`.

Reply:
It now says `(rename f p).support = finset.image (map_domain f) p.support`